### PR TITLE
Attempt to fix Windows builds

### PR DESCRIPTION
**Pull request by drdavella**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ environment:
     NP_BUILD_DEP: "numpy==1.10.4"
     NP_TEST_DEP: "numpy==1.10.4"
     OTHER_BUILD_DEP: "cython jinja2"
-    OTHER_TEST_DEP: "pytest-astropy pytest<3.7"
+    OTHER_TEST_DEP: "pytest-astropy \"pytest<3.7\""
```
Comment (bsipocz) [inline appveyor.yml:12]: wow, this was a stupid mistake of mine, thanks

**Comment by bsipocz** [comment]
@drdavella - I wonder how the merge commit ended up here, but this repos is a mess anyway, so let's merge if it passes.

**Comment by drdavella** [comment]
@bsipocz yeah I think I messed it up. I'll try to push a clean history once CI is done.

**Comment by drdavella** [comment]
@bsipocz I wonder if we could restart the appveyor build? I think the failure is really just a time out.

**Comment by bsipocz** [comment]
This seems to solve some of the issues even with the time outs so I'm merging and see what happens.

**Comment by bsipocz** [comment]
Thanks @drdavella!

**Pull request merged**